src/@components/atoms/stories.tsx

```diff
@@ -0,0 +1,11 @@
+import { storiesOf } from "@storybook/react";
+import React from "react";
+
+import { Button } from ".";
+
+storiesOf("@components/atoms", module).add("Button", () => (
+  <>
+    <Button>Sample Button</Button>
+    <Button secondary>Secondary Button</Button>
+  </>
+));
```

Comment:
Two things..

First I would use such structuring of stories:

```diff
diff --git a/src/@components/atoms/stories.tsx b/src/@components/atoms/stories.tsx
index 7e9635b..9eadfb1 100644
--- a/src/@components/atoms/stories.tsx
+++ b/src/@components/atoms/stories.tsx
@@ -3,9 +3,6 @@ import React from "react";
 
 import { Button } from ".";
 
-storiesOf("@components/atoms", module).add("Button", () => (
-  <>
-    <Button>Sample Button</Button>
-    <Button secondary>Secondary Button</Button>
-  </>
-));
+storiesOf("@components / atoms / Button", module)
+  .add("primary", () => <Button>Sample Button</Button>)
+  .add("secondary", () => <Button secondary>Secondary Button</Button>);
```

Second thought -> I think that stories.tsx should be placed on root level of components dir because soon we will have here dozens of components and the file will grow enormously :). Or lets have ```@components/atoms/stories/<component>.tsx``` structure :)

Reply:
Moved `stories.tsx` into component dir with a helper decorator for creating stories